experiment/invitations-accepter/main_test.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+
+	"k8s.io/test-infra/prow/github"
+	"k8s.io/test-infra/prow/github/fakegithub"
+)
+
+func TestAcceptInvitations(t *testing.T) {
+
+	testCases := []struct {
+		id          string
+		invitations map[int]github.UserRepoInvitation
+		expected    []github.UserRepoInvitation
```

Comment:
I don't actually see any expected invitations in any tests, am I missing something?

Reply:
Well, the expected should be empty if the tests succeed.

Reply:
So why make it an arg to the testcases?

Reply:
The only reason was to indicate that this was the expected value. I will remove it. Thanks